docs/core/build/emulator.md

```diff
@@ -1,5 +1,8 @@
 # Build instructions for Emulator (Unix port)
 
+> :bulb: **Hint**:
+Using emulator as described here is useful during firmware development. If you intend to use only released versions of firmware, you might be looking for [Trezor User Env](https://github.com/trezor/trezor-user-env/tree/master).
```

Comment:
> If you intend to use only released versions of firmware
you can also use fw from a branch in trezor-user-env using a gitlab link, no? But maybe only internally?🤔

Reply:
You're right. I reworded to 

`If you intend to use the emulator without modifying the firmware, you ....`